mmv1/products/filestore/Instance.yaml

```diff
@@ -209,6 +209,12 @@ properties:
                   An integer representing the anonymous group id with a default value of 65534.
                   Anon_gid may only be set with squashMode of ROOT_SQUASH. An error will be returned
                   if this field is specified for other squashMode settings.
+              - name: 'network'
```

Comment:
Is this field mutable?

Reply:
Yes

Reply:
Added a unit test to validate updates of the network field